docs/decisions/006-homogeneous-build-across-environments.md

```diff
@@ -0,0 +1,22 @@
+# 6. Homogeneous build across environments
```

Comment:
Well, to me this decisions sounds like: Use Docker :-D

Reply:
I don't know why - some things seem to fit together 😄